data/mods/Defense_Mode/mapgen/mansion.json

```diff
@@ -368,6 +360,58 @@
       "place_loot": [ { "item": "lawn_dart", "x": [ 7, 16 ], "y": [ 18, 22 ], "chance": 40, "repeat": [ 1, 2 ] } ]
     }
   },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "dm_mansion_t2d" ],
+    "weight": 1000,
```

Comment:
Should this weight field be getting removed if it's just the default val?

Reply:
"(if there are variants with one having default weight it's still included so the difference can be seen more easily at a glance)"
I appreciate that field is doing nothing it just helps users not familiar with the default to see the relative weights and for anyone to easier identify it as a om_terrain id with variants without having to rely on noticing the other map(s)